drivers/marvell/uart/a3700_console.S

```diff
@@ -5,16 +5,26 @@
  * https://spdx.org/licenses
  */
 
+#include <arch.h>
 #include <asm_macros.S>
 #include <a3700_console.h>
```

Comment:
Actually, now that I'm taking a look at https://github.com/ARM-software/arm-trusted-firmware/pull/1683, you are missing ``#define USE_FINISH_CONSOLE_REG_2`` before the include of ``console_macros.S``.

Reply:
After adding this define I do not see any output from TF-A BLx anymore.

Reply:
Old output:
========
TIM-1.0
WTMI-devel-18.07.0-6050fd5
WTMI: system early-init
CPU VDD voltage default value: 1.155V
NOTICE:  Booting Trusted Firmware
NOTICE:  BL1: v2.0(release):v2.0-257-g036a759 (Marvell-devel-18.09.1)
NOTICE:  BL1: Built : 12:21:02, Nov 15 2018
NOTICE:  BL1: Booting BL2
NOTICE:  BL2: v2.0(release):v2.0-257-g036a759 (Marvell-devel-18.09.1)
NOTICE:  BL2: Built : 12:21:03, Nov 15 2018
NOTICE:  BL1: Booting BL31
NOTICE:  BL31: v2.0(release):v2.0-257-g036a759 (Marvell-devel-18.09.1)
NOTICE:  BL31: Built : 12:2

U-Boot 2018.03-devel-18.11.0-00290-g0e283d5-dirty (Nov 15 2018 - 12:20:01 +0200)

Model: Marvell Armada 3720 Community Board ESPRESSOBin
       CPU    @ 1000 [MHz]
       L2     @ 800 [MHz]
       TClock @ 200 [MHz]
       DDR    @ 800 [MHz]
DRAM:  1 GiB
Comphy chip #0:
Comphy-0: USB3          5 Gbps    
Comphy-1: PEX0          2.5 Gbps  
Comphy-2: SATA0         6 Gbps    
SATA link 0 timeout.

new output:
=========
TIM-1.0
WTMI-devel-18.07.0-6050fd5
WTMI: system early-init
CPU VDD voltage default value: 1.155V


U-Boot 2018.03-devel-18.11.0-00290-g0e283d5-dirty (Nov 15 2018 - 12:20:01 +0200)

Model: Marvell Armada 3720 Community Board ESPRESSOBin
       CPU    @ 1000 [MHz]
       L2     @ 800 [MHz]
       TClock @ 200 [MHz]
       DDR    @ 800 [MHz]
DRAM:  1 GiB
Comphy chip #0:
Comphy-0: USB3          5 Gbps    
Comphy-1: PEX0          2.5 Gbps  
Comphy-2: SATA0         6 Gbps    
SATA link 0 timeout.

Reply:
Ignore my complain, I see what the problem - the driver registration macro has changed.